packages/block-editor/src/hooks/gap.js

```diff
@@ -54,8 +55,12 @@ export function getGapBoxControlValueFromStyle( blockGapValue ) {
 
 	const isValueString = typeof blockGapValue === 'string';
 	return {
-		top: isValueString ? blockGapValue : blockGapValue?.top,
-		left: isValueString ? blockGapValue : blockGapValue?.left,
+		top: isValueString
+			? getSpacingPresetCssVar( blockGapValue )
```

Comment:
Do the existing tests for `getGapBoxControlValueFromStyle` cover these changes already?

Reply:
Oh, good call — I'll look into adding some tests 🙂